charts/cloudserver-front/values.yaml

```diff
@@ -11,6 +11,9 @@ credentials:
   keyId: deployment-specific-access-key
   secretKey: deployment-specific-secret-key
 
+# Logging options: info/debug/trace
+logLevel: info
```

Comment:
Sorry for nit-pick'ing, but now I wonder whether we shouldn't use
```yaml
logging:
  # Options: info, debug, trace
  level: info
```
to allow for future extensions, and use similar configuration mechanisms for services with more 'complex' logging features.

On the other hand, there seems to be no common convention in https://github.com/kubernetes/charts/

Reply:
Makes perfect sense. I will update this.